cryptographic_estimators/SDEstimator/SDAlgorithms/ball_collision.py

```diff
@@ -66,80 +71,97 @@ def __init__(self, problem: SDProblem, **kwargs):
     @optimal_parameter
     def l(self):
         """
-        Return the optimal parameter $l$ used in the algorithm optimization
+        Return the optimal parameter $l$ used in the algorithm optimization.
 
-        EXAMPLES::
-
-            sage: from cryptographic_estimators.SDEstimator.SDAlgorithms import BallCollision
-            sage: from cryptographic_estimators.SDEstimator import SDProblem
-            sage: A = BallCollision(SDProblem(n=100,k=50,w=10))
-            sage: A.l()
+        Examples:
```

Comment:
how does this napoleon extension for sphincs work? standard html doc generation via sphincs required capital spelling of EXAMPLES and TESTS and a double colon followed by a blank line? Maybe we should investigate this first to avoid later changes?